caom2-search-server/src/main/resources/META-INF/resources/js/cadc.search.app.js

```diff
@@ -1160,16 +1209,17 @@
                   activeSearchForm.setSelectValue(ca.nrc.cadc.search.CAOM2_TARGET_NAME_FIELD_ID, qKey,
                                                   decodeURIComponent(qValue.join()));
                 }
-                else
+                else if (qKey !== "useMaq")
                 {
+                  // useMaq has been handled prior to the data train being loaded
                   activeSearchForm.setInputValue(qKey, decodeURIComponent(qValue.join()));
                 }
 
                 doSubmit = true;
               }
             });
 
-            $submitForm.find("input").change();
+            //$submitForm.find("input").change();
```

Comment:
This used to force a check to react to input value changes and make sure that units were converted, for example.  Is it replaced elsewhere?